sys/conf/kern.pre.mk

```diff
@@ -142,6 +142,10 @@ CFLAGS+=	${GCOV_CFLAGS}
 # can override the others.
 CFLAGS+=	${CONF_CFLAGS}
 
+.if ${MACHINE_ARCH:Maarch64*c*}
+LDFLAGS+=	--morello-c64-plt
```

Comment:
Ditto re XXX

Reply:
This is possibly deprecated after my last change in morello-lld. But needs to land first.

Reply:
Yeah let's keep it for now and deal with all the LDFLAGS hacks in one go once they fix their toolchain to do things properly